controllers/module_nmc_reconciler.go

```diff
@@ -188,33 +190,67 @@ func (mnrh *moduleNMCReconcilerHelper) finalizeModule(ctx context.Context, mod *
 	return mnrh.client.Patch(ctx, mod, client.MergeFrom(modCopy))
 }
 
-func (mnrh *moduleNMCReconcilerHelper) getNodesList(ctx context.Context) ([]v1.Node, error) {
-	nodes := v1.NodeList{}
-	err := mnrh.client.List(ctx, &nodes)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get list of nodes: %v", err)
-	}
-	return nodes.Items, nil
-}
+func (mnrh *moduleNMCReconcilerHelper) getNodesListBySelector(ctx context.Context, mod *kmmv1beta1.Module) ([]v1.Node, error) {
```

Comment:
I prefer to use this name, since it is the same function as used in other controllers, and it will help to easily identify it when we start working on refactoring for v2.1

Reply:
I think it is confusing 🤷🏻‍♂️ but will live with it if you insist.